packages/block-events-stream/README.md

```diff
@@ -0,0 +1,34 @@
+# Block events stream
+
+## Example
+
+```ts
+import {
+  createBlockStream,
+  isNonPendingBlock,
+  groupLogsByBlockNumber,
+  blockRangeToLogs,
+} from "@latticexyz/block-events-stream";
+
+const latestBlock$ = await createBlockStream({ publicClient, blockTag: "latest" });
+
+const latestBlockNumber$ = latestBlock$.pipe(
+  filter(isNonPendingBlock),
+  map((block) => block.number)
+);
+
+latestBlockNumber$
+  .pipe(
+    map((latestBlockNumber) => ({ startBlock: 0n, endBlock: latestBlockNumber })),
+    blockRangeToLogs({
+      publicClient,
+      address,
+      events,
```

Comment:
is it worth adding the definitions of these to the example? or a comment about how to define them?